.github/workflows/unittests.yml

```diff
@@ -27,7 +28,6 @@ jobs:
       - name: Setup joinmarket + virtualenv
         if: steps.cache-venv.outputs.cache-hit != 'true'
         run: |
-          sudo apt update
           ./install.sh --develop --with-qt
```

Comment:
BTW, what is the reason we do `--with-qt` here? To test all dependencies install correctly? Should we also do `--with-local-tor` for the same reason?

Reply:
I don't know/remember who included that, but that would be my reasoning. You could argue to remove both though, on the basis that we only *require* that the application can build in its basic form. I'm happy for you to make that judgement call.